utils/TestUtils.ts

```diff
@@ -98,20 +106,17 @@ export class RedisCluster {
                 }
             }
 
-            console.log(command);
```

Comment:
Removed these console.log() statement. Not sure if this is something that we want to keep. If there is a specific reason to log the command, I can revert this change.